src/main/resources/templates/gardens/view.html

```diff
@@ -29,7 +31,8 @@ <h4>Owner </h4>
         </div>
     </div>
     <div class = "row  justify-content-center">
-        <a th:href="@{/user/gardens/list}">Return to Water Gardens</a>
+        <a class="btn btn-primary" th:href="@{/user/gardens/list}">Return to Water Gardens</a>
```

Comment:
I thought these buttons looked a little tight side-by-side, what are your thoughts on adding style="margin-right:10px;" to this first button to give them some space?

Reply:
Yeah, there’s definitely room for more styling in several areas.

Reply:
I added some margins with several other buttons, too.